src/components/sidenav/sidenav.js

```diff
@@ -365,6 +367,8 @@ function SidenavDirective($mdMedia, $mdUtil, $mdConstant, $mdTheming, $animate,
       ]).then(function() {
         // Perform focus when animations are ALL done...
         if (scope.isOpen) {
+          // Notifies child components that the sidenav was opened.
+          ngWindow.triggerHandler('resize');
```

Comment:
True, however we can assume that all the handlers are registered on the window, because the "resize" event doesn't work on anything else.